app/main.py

```diff
@@ -1,3 +1,11 @@
 def cache(func):
-    # Write your code here
-    pass
+    cache_date = []
+
+    def wrapper(*args, **kwargs):
+        if func(*args, **kwargs) in cache_date:
+            print("Getting from cache")
+        else:
+            print("Calculating new result")
+        cache_date.append(func(*args, **kwargs))
+        return func(*args, **kwargs)
```

Comment:
Here you call the function a second time. The first time you call it in the 9th line.
It's better you use the already calculated value from the cache_date